content/blog/2024/01/2024-01-29-nominate-someone-2024-jenkins-contributor awards.adoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Nominate Someone – 2024 Jenkins Contributor Awards"
+tag:
+- awards
+- jenkins
+- community
+author: alyssat
+description: "Brief summary to nominate Jenkins community members for awards"
+opengraph: 
+  image: /images/post-images/2024/community awards 2024 jenkins nominate 1.png
+---
+
+image:/images/post-images/2024/community awards 2024 jenkins nominate 1.png[Nominate Someone – 2024 Jenkins Contributor Awards]
```

Comment:
Please avoid spaces in filenames, they're source of many potential problems when programmatically interacting with them.

Reply:
The deploy preview has these as broken images:

<img width="583" alt="Screenshot 2024-01-29 at 9 05 27 AM" src="https://github.com/jenkins-infra/jenkins.io/assets/99040580/b62c991b-22f6-4fd3-9bba-a27579588c35">

<img width="580" alt="Screenshot 2024-01-29 at 9 05 32 AM" src="https://github.com/jenkins-infra/jenkins.io/assets/99040580/6ea35682-32c2-4635-b868-77f678c06f30">

Reply:
got this squared away with renaming the file on upload.